src/http/matcher/mod.rs

```diff
@@ -29,24 +36,26 @@ mod header;
 #[doc(inline)]
 pub use header::HeaderMatcher;
 
-use crate::{
-    http::Request,
-    service::{context::Extensions, matcher::IteratorMatcherExt, Context},
-    stream::matcher::SocketMatcher,
-};
-
-#[derive(Debug, Clone)]
+#[derive(Debug)]
```

Comment:
this is to be manually implemented as well, as otherwise `State` and `Body` are expected to be debug, which isn't the case